### ISSUE TITLE
Perfetto attempting to access internal google URL to download scripts on non-corp machine.

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/devtools#6006]
When running a flutter application on a non-corp machine, I'm unable to load the perfetto viewer. Looking in the browser network tab shows a failed request to https://storage.cloud.google.com/perfetto-ui-internal/is_internal_user.js .

Is there some internal configuration option I might have enabled accidentally?

**Comment by CoderDake** [comment]
@kenzieschmoll any idea whether this could prevent perfetto from loading?

**Comment by kenzieschmoll** [comment]
That request looks like it is coming from perfetto itself, not from DevTools. Can you open ui.perfetto.dev and if so do you see that same request? Another thing to try - try opening DevTools in an incognito browser or clearing your browser cache.

**Comment by jonahwilliams** [comment]
Yes, I see the same issue on ui.perfetto.dev. If I access either that or devtools via incognito, instead it fails on a google sign in request:

https://accounts.google.com/v3/signin/identifier?...

With continue urls that point to the same resource.

**Comment by kenzieschmoll** [comment]
We should file an issue here, since this is not specific to the DevTools configuration: https://github.com/google/perfetto/issues

**Comment by srawlins** [comment]
Closing as an external/upstream issue: This was confirmed to be an upstream issue originating from Perfetto (ui.perfetto.dev) rather than DevTools configuration. Updated builds of compiled Perfetto assets have since been rolled into DevTools.

If there is more work to do here, please let us know by filing a new issue with up to date information. Thanks!
